<h1>Class yii\web\MethodNotAllowedHttpException</h1> <div class="top-nav"> <a href="index.html">All Classes</a> | <a href="#properties">Properties</a> | <a href="#methods">Methods</a> </div> <table> <colgroup> <col class="col-name"> <col class="col-value"> </colgroup> <tr>
<th>Inheritance</th>
<td>
<a href="yii-web-methodnotallowedhttpexception.html">yii\web\MethodNotAllowedHttpException</a> » <a href="yii-web-httpexception.html">yii\web\HttpException</a> » <a href="yii-base-userexception.html">yii\base\UserException</a> » <a href="yii-base-exception.html">yii\base\Exception</a> » <a href="http://www.php.net/class.exception">Exception</a>
</td>
</tr> <tr>
<th>Available since version</th>
<td>2.0</td>
</tr> <tr> <th>Source Code</th> <td><a href="https://github.com/yiisoft/yii2/blob/master/framework/web/MethodNotAllowedHttpException.php">https://github.com/yiisoft/yii2/blob/master/framework/web/MethodNotAllowedHttpException.php</a></td> </tr> </table>  <p><strong>MethodNotAllowedHttpException represents a "Method Not Allowed" HTTP exception with status code 405.</strong></p>    <h2 id="properties">Public Properties</h2>  <table> <colgroup> <col class="col-property"> <col class="col-type"> <col class="col-description"> <col class="col-defined"> </colgroup> <tr> <th>Property</th>
<th>Type</th>
<th>Description</th>
<th>Defined By</th> </tr> <tr id="$statusCode"> <td><a href="yii-web-httpexception.html#%24statusCode-detail">$statusCode</a></td> <td><a href="http://www.php.net/language.types.integer">integer</a></td> <td>HTTP status code, such as 403, 404, 500, etc.</td> <td><a href="yii-web-httpexception.html">yii\web\HttpException</a></td> </tr> </table>    <h2 id="methods">Public Methods</h2>  <table> <colgroup> <col class="col-method"> <col class="col-description"> <col class="col-defined"> </colgroup> <tr> <th>Method</th>
<th>Description</th>
<th>Defined By</th> </tr> <tr id="__construct()"> <td><a href="yii-web-methodnotallowedhttpexception.html#__construct()-detail">__construct()</a></td> <td>Constructor.</td> <td><a href="yii-web-methodnotallowedhttpexception.html">yii\web\MethodNotAllowedHttpException</a></td> </tr> <tr id="getName()"> <td><a href="yii-web-httpexception.html#getName()-detail">getName()</a></td> <td></td> <td><a href="yii-web-httpexception.html">yii\web\HttpException</a></td> </tr> </table>    <h2 id="constants">Method Details</h2> <div class="method-doc"> <h3 id="__construct()-detail"> __construct() <span class="detail-header-tag small"> public method </span> </h3>  <p><strong>Constructor.</strong></p>  <table> <tr><th colspan="3">public void <strong><a href="yii-web-methodnotallowedhttpexception.html#__construct()-detail">__construct</a></strong> ( $message = null, $code = 0, <a href="http://www.php.net/class.exception">Exception</a> $previous = null )</th></tr> <tr> <td>$message</td> <td><a href="http://www.php.net/language.types.string">string</a></td> <td>
<p>Error message</p> </td> </tr> <tr> <td>$code</td> <td><a href="http://www.php.net/language.types.integer">integer</a></td> <td>
<p>Error code</p> </td> </tr> <tr> <td>$previous</td> <td><a href="http://www.php.net/class.exception">Exception</a></td> <td>
<p>The previous exception used for the exception chaining.</p> </td> </tr> </table>  </div><div class="_attribution">
  <p class="_attribution-p">
    &copy; 2008&ndash;2017 by Yii Software LLC<br>Licensed under the three clause BSD license.<br>
    <a href="http://www.yiiframework.com/doc-2.0/yii-web-methodnotallowedhttpexception.html" class="_attribution-link">http://www.yiiframework.com/doc-2.0/yii-web-methodnotallowedhttpexception.html</a>
  </p>
</div>
